src/Uno.UWP/ApplicationModel/DataTransfer/DataPackage.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Windows.ApplicationModel.DataTransfer
+{
+	public partial class DataPackage
+	{
+		internal string Text { get; private set; }
+
+#if !(NET46 || __MACOS__)
```

Comment:
For `NET46`, clipboard access is usually done through the `Clipboard` class that you would include either from wpf or winforms. I am not sure if it would be a good idea to dynamically check the referenced library to determine which implementation to invoke (dynamically) in the case of an wpf/winform project. Or, which library to import in the case of a library/console project.

As for `__MACOS__`, I couldn't simply find an environment to test it.